MobileApplication/MobileApplication/appsettings.json

```diff
@@ -1,3 +1,3 @@
 {
-  "ApiBaseAddress": "http://192.168.1.54:45455/"
+  "ApiBaseAddress": "http://192.168.8.106:45459/"
```

Comment:
It's not the best practise to edit appsettings after each pull. User secrets may be a solution but I'm not sure yet.